exporter.go

```diff
@@ -87,23 +96,28 @@ func NewCounterContainer() *CounterContainer {
 }
 
 func (c *CounterContainer) Get(metricName string, labels prometheus.Labels, help string) (prometheus.Counter, error) {
-	counterVec, ok := c.Elements[metricName]
+	labelNames := getLabelNames(labels)
+	mapKey := metricName + "," + strings.Join(labelNames, ",")
```

Comment:
you're doing this in several places, please break it out into a function so that we can be sure it's consistent